Sharity/sharity/notification/migrations/0003_auto_20220607_2242.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.13 on 2022-06-07 22:42
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('notification', '0002_alter_notification_serviceid'),
+    ]
+
+    operations = [
```

Comment:
@ossarioglu Could you check this migration file? System created this when I execute the `makemigrations` command. I do not want to affect the last fixes in a wrong way.

Reply:
@gokhanustuner   This migration is OK. It's about changing notification character limit to 1000 characters